src/core/mount.js

```diff
@@ -74,6 +75,12 @@ function reset() {
 }
 
 function init() {
+  document.addEventListener("keydown", (e) => {
+    if (e.key === "Escape") {
+      store.dispatch(closeLastModal());
+    }
+  });
+
```

Comment:
```suggestion
function init() {
  // We only want to close the single last modal when the user hits escape.
  // Consequently we only want a single event listener. We cannot guarantee this
  // on the app (or component) level as a page may contain multiple apps. 
  // init provides a single entry point for loading all apps and is suitable for our
  // purpose.
  document.addEventListener("keydown", (e) => {
    if (e.key === "Escape") {
      store.dispatch(closeLastModal());
    }
  });

```

We agree on the solution but I think this change warrants some documentation/inline code comment about why it is needed and why it is placed at this level in the codebase.